packages/web/components/Player/Section/PlayerHero.tsx

```diff
@@ -21,16 +22,26 @@ export const PlayerHero: React.FC<Props> = ({ player }) => {
   const [show, setShow] = React.useState(
     getPlayerDescription(player).length < BIO_LENGTH,
   );
-
+  const avatar = hasPlayerImage(player) ? (
+    <Avatar
+      w={{ base: '32', md: '56' }}
+      h={{ base: '32', md: '56' }}
+      src={getPlayerImage(player)}
+      name={getPlayerName(player)}
+      bg=""
```

Comment:
Why can't you set `bg=""` always ?

If you can't do this, better to write something like 
`bg={hasPlayerImage(player)) ? "transparent" : undefined}`
So you don't repeat the same component twice

Reply:
That was my initial thought, but as soon as you pass in bg it will put it to transparent, only if you don't pass it in it colors it. I researched a bit into why, but it seems to be a strange Chakra UI behavior.

Reply:
transparent is not working with chakra?

Reply:
It seems like a bug, as soon as you pass in `bg` it will make it transparent. Even if you pass in `bg="yellow"`

Reply:
That's why the little cheeky workaround. I couldn't find a way to conditionally pass the bg prop or not at all.

Reply:
As I wrote if you pass `undefined` it will do like you didn't pass a prop
`bg={hasPlayerImage(player)) ? "" : undefined}`

Reply:
That was my logic as well, yet it does not work :)
It just makes it all transparent